dazel/test/projects/codegen_consumer/pubspec.lock

```diff
@@ -0,0 +1,11 @@
+# Generated by pub
```

Comment:
oops, yeah it's already in .gitignore but I force added it because I thought it was necessary for the test. Removed again